x-pack/legacy/plugins/canvas/canvas_plugin_src/functions/common/index.ts

```diff
@@ -47,9 +47,10 @@ import { rounddate } from './rounddate';
 import { rowCount } from './rowCount';
 import { repeatImage } from './repeatImage';
 import { revealImage } from './revealImage';
-import { savedMap } from './saved_map';
-import { savedSearch } from './saved_search';
-import { savedVisualization } from './saved_visualization';
+// TODO: Re-Enable once we get filtering from embeddables figured out
```

Comment:
nit: refer to the Github issue in the comment... if there isn't one, let's create one.